ui/app/templates/evaluations/index.hbs

```diff
@@ -12,6 +12,7 @@
         />
       </div>
       <div class="toolbar-item is-right-aligned">
+        <!-- TODO: add support for status and triggered by
```

Comment:
`{{!--` as a comment style in .hbs will prevent it from showing up in rendered output -- either way is totally fine, but if we don't want this rendered in built source, you could switch your `<` for a `{{` etc.

Reply:
Yeah, thanks! The linter screamed at me for using the wrong one 😄